dev/bots/test.dart

```diff
@@ -176,6 +178,55 @@ Future<bq.BigqueryApi> _getBigqueryApi() async {
   }
 }
 
+List<List<String>> _partitionToolTests() {
+  final List<File> pending = <File>[];
+  final String toolTestDir = path.join(toolRoot, 'test');
+  for (FileSystemEntity entity in Directory(toolTestDir).listSync(recursive: true)) {
+    if (entity is File && entity.path.endsWith('_test.dart')) {
+      pending.add(entity);
+    }
+  }
+  final List<String> groupA = <String>[];
+  final List<String> groupB = <String>[];
+  for (int i = 0; i < pending.length; i += 1) {
```

Comment:
out of curiosity: why `i += 1` instead of `i++`?

Reply:
I don't think it matters TBH, but I generally prefer this to avoid any comments over pre vs post incrementing.